### ISSUE TITLE
X-Axis Legend at timeseries charts is missing

### ISSUE THREAD

**Issue by gk998** [ntop/ntopng#6146]
I'm new to ntopng and I run community 5.0.211124 on FreeBSD 12.1. I noticed that the timeseries charts for interfaces seem to be cut. I can hardly see the upper half of the "0" and can't see the legend with the timestamps. 
Is this intended?
<img width="625" alt="grafik" src="https://user-images.githubusercontent.com/29233091/146644894-f9947fc9-4cc4-4751-9db2-f3f98a8793f7.png">

**Comment by MatteoBiscosi** [comment]
Hi @gk998 I fixed the 0 cut, but this is the maximum i can do right now, these are old js libraries and are not really flexible. In the near future however we are planning on changing these libraries. Regarding the time (x axis missing), I added the time on hover.
![Screenshot from 2021-12-20 13-48-13](https://user-images.githubusercontent.com/49585191/146769511-bf0c69f4-dd1d-4404-8047-8b52a794c768.png)
